team-config.yml

```diff
@@ -382,3 +382,13 @@ crime-platform:
   tags:
     application: "core"
     businessArea: "Crime"
+
+aad-domain-services:
+  team: "platform operations" 
+  namespace: ""
+  slack:
+    contact_channel: "#platops-help" 
+    build_notices_ch"nnel: ""
+  tags:
+    application: "core"
+    businessArea: "CFT"
```

Comment:
```suggestion
    businessArea: "Cross-Cutting"
```

Reply:
Definitely Cross-Cutting not used by CFT at all.

Reply:
https://github.com/hmcts/dtspo-aad-domain-services/blob/6b46122b8cf9722a52ba14ed8313966714fb0a4b/terraform/02-inputs-optional.tf#L13